packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts

```diff
@@ -1434,3 +1437,93 @@ describe('attachWebAclId', () => {
   });
 
 });
+
+test('Selection criteria does set Media Quality Based failover for Origin Group', () => {
```

Comment:
Could we move these tests to `origin-group.test.ts` instead? I was going to ask to add unit tests for origin group but I think it would be redundant with these.

Reply:
fixed